.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here. For example, "Fixes #123", "Related to #456", or "Addresses #789".
```

Comment:
GitHub has a feature [Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) that is:

* Link PR and issue automatically by using specific keywords.
* If PR is closed, GitHub automatically closes the linked issues.

Regarding this feature, `Fixes #123` is one of the keywords (you can see the list of keywords in the above linked document). So, if we use `Fixes`, GitHub links and closes related issues automatically.

However, `Related to #456` and `Addresses #789` are not keywords of this feature. So, if we use these keywords, GitHub does not link and close related issues automatically.

In other words, the behavior will be different depending on `which word we will use`. And, there is no description of this GitHub's linking feature in this template.

So, I think it causes a bit of confusion.

I think it would be better to:

* Describe (add link) the GitHub's linking feature in this template.
* Do NOT use `Fixes` that invoke the GitHubs' linking feature.
* Use other keywords instead of `Related to` or `Address`, to invoke GitHub's linking feature always.

What do you think?

Reply:
Thank you for sharing this. I didn't know that GitHub had this feature.

What do you think about simplifying this description as follows?:

```suggestion
## Related issues and/or PRs

> If this PR addresses or references any issues and/or other PRs, list them here.
```

Reply:
> What do you think about simplifying this description as follows?

Thank you for the update!
This simplified description looks good to me!